integration-tests/testconfig/ccip/ccip.toml

```diff
@@ -146,4 +147,8 @@ addresses_to_fund = [
 ]
 
 [CCIP.PrivateEthereumNetworks.SIMULATED_2.EthereumChainConfig.HardForkEpochs]
-Deneb = 500
\ No newline at end of file
+Deneb = 500
```

Comment:
we can remove that whole section, we are using a newer version of genesis generator and Deneb is already in the past :-)